platform-sdk/swirlds-base/src/main/java/com/swirlds/base/function/CheckedConsumer.java

```diff
@@ -48,7 +48,7 @@
      */
     @NonNull
     static <T, E extends Exception> CheckedConsumer<T, E> of(@NonNull final Consumer<T> consumer) {
-        ArgumentUtils.throwArgNull(consumer, "consumer");
+        Objects.requireNonNull(consumer, "consumer must not be null");
```

Comment:
Not sure if the String second is still needed? Or could we do that easier?

I always used without the second parameter.